src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarMetadata.tsx

```diff
@@ -18,6 +18,13 @@ export class ArtworkSidebarMetadata extends React.Component<
     const { artwork } = this.props
     return (
       <Box>
+        {artwork.is_biddable &&
+          artwork.sale_artwork &&
+          artwork.sale_artwork.lot_label && (
```

Comment:
Maybe: 
```js
const lotLabel = get(artwork, p => p.is_biddable && p.artwork.sale_artwork.lot_label)

{lotLabel && 
 ...
}
```
? 

Basically the same, but i think using `get` is a little less error prone.

Reply:
I guess get is a bit confusing here because of how we need to check for `is_biddable` and not only to unpack `p.artwork.sale_artwork.lot_label`. But happy to change if you think it's less error prone.

Reply:
It's debatable! no strong prefs at all.

Reply:
I guess setting it to const eliminates long repeat inside the clause - so you win:)